compiler/parser.mly

```diff
@@ -83,7 +83,7 @@ array_literal:
 
 primitive_literal_list:
     /* Nothing */         { [] }
-  | primitive_literal_list COMMA primitive_literal    { $3 :: $1 }
+  | primitive_literal SEMICOLON primitive_literal_list   { $1 :: $3 }
```

Comment:
Just thought it looked better, cuz the way I have it in there is `[3;4;5;6;1;]`, so there's one at the end - comma looked weird. This will likely get changed though